src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -41,9 +43,23 @@ public String execute(String commandText) throws CommandException, ParseExceptio
         String commandResult = command.execute(model);
         storage.saveAddressBook(model.getAddressBook());
 
+        // keep track of valid commands
+        commandHistory.add(commandText);
+        logger.info("\"" + commandText + "\" pushed to stack, commandTextFuture stack cleared");
```

Comment:
Would it be better for this to be:
```
logger.info("\"" + commandText + "\" added to commandHistory");
```
instead?

Reply:
Yes, it is more succinct.